Encrypted Ink/Screens/AccountsListViewController.swift

```diff
@@ -169,6 +174,56 @@ class AccountsListViewController: NSViewController {
         }
     }
     
+    @objc private func didClickImportFromMetamask() {
```

Comment:
Move implementation to a separate file.
AccountsViewController does not have to know so much about MetaMask.